Scripts/Modules/TooltipModule/TooltipModule.cs

```diff
@@ -91,21 +94,28 @@ void Update()
 
 				if (!IsValidTooltip(tooltip))
 					m_TooltipsToHide.Add(tooltip);
+
+				if (tooltipData.persistent)
+				{
+					var duration = tooltipData.duration;
+					if (duration > 0 && Time.time - tooltipData.lastModifiedTime + k_Delay > duration)
+						m_TooltipsToHide.Add(tooltip);
+				}
 			}
 
 			foreach (var tooltip in m_TooltipsToHide)
 			{
-				HideTooltip(tooltip);
+				HideTooltip(tooltip, true);
 			}
 		}
 
 		static Transform GetTooltipTarget(ITooltip tooltip)
 		{
 			var placement = tooltip as ITooltipPlacement;
-			var target = ((MonoBehaviour)tooltip).transform;
 			if (placement != null)
-				target = placement.tooltipTarget;
-			return target;
+				return placement.tooltipTarget;
+
+			return ((MonoBehaviour)tooltip).transform;
```

Comment:
Yes. This is why I made this change in the first place, because I ended up trying a non-MB tooltip. But in this case you would never be able to use something that was just ITooltip and not a monobehaviour. I'd want some sort of error in this case, rather than the system silently doing nothing.